README.md

```diff
@@ -108,5 +113,11 @@ Your answer here.
 How is `git pull` related to `git fetch`?
 
 ```text
-Your answer here.
+
+A git pull is what you would do to bring a local branch up-to-date with its remote version, while also updating your other remote-tracking branches.
+
+You can do a git fetch at any time to update your remote-tracking branches. This operation never changes any of your own local branches, and is safe to do without changing your working copy.
+
```

Comment:
http://stackoverflow.com/questions/292357/what-are-the-differences-between-git-pull-and-git-fetch

Please put it in your own words!